packages/pyright-scip/package.json

```diff
@@ -7,8 +7,8 @@
     "build": "webpack --mode production --progress",
     "clean": "shx rm -rf ./dist ./out README.md LICENSE.txt",
     "prepack": "npm run clean && shx cp ../../README.md . && shx cp ../../LICENSE.txt . && npm run build",
-    "check-snapshots-new": "uvu ./test -r ts-node/register --ignore dist --check",
-    "update-snapshots-new": "uvu ./test -r ts-node/register --ignore dist",
+    "check-snapshots-new": "uvu ./test -r ts-node-transpile-only/register --check",
+    "update-snapshots-new": "uvu ./test -r ts-node-transpile-only/register --ignore dist",
```

Comment:
Do not spend time on type-checking while running tests

Reply:
Hmm, does this require updating `package.json`? I can't tell which package this comes from... maybe it's being picked up from global state on your machine?

Reply:
If this is not strictly necessary, I'm fine with sticking to `ts-node`